Tests/L0/Maven/_suite.ts

```diff
@@ -1,6 +1,5 @@
 /// <reference path="../../../definitions/mocha.d.ts"/>
 /// <reference path="../../../definitions/node.d.ts"/>
-/// <reference path='../../../Tasks/Maven/pmdForMaven.ts'/>
```

Comment:
These ../.. references cause failures in copying test resources and had to be removed. This has led to the removal of unit tests, so integration testing will need to pick up the slack here.